.github/workflows/validations.yaml

```diff
@@ -49,12 +49,15 @@ jobs:
 
   Integration-Test:
     name: "Integration tests"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
note: ubuntu 22.04 was required to install `rootlesskit` for the rootless containerd setup, so I bumped this everywhere to be consistent